### ISSUE TITLE
Issue with "content": Accessibility concerns is outdated

### ISSUE THREAD

**Issue by digi-booster** [mdn/content#9284]
MDN URL: https://developer.mozilla.org/en-US/docs/Web/CSS/content

#### What information was incorrect, unhelpful, or incomplete?

Certain Assistive technology/browser combinations will not announce CSS-generated content.
#### Specific section or headline?

[Content > Accessibility concerns](https://developer.mozilla.org/en-US/docs/Web/CSS/content#accessibility_concerns)
#### What did you expect to see?
Assistive technology with IE browser will not announce it.

#### Did you test this? If so, how?

[CSS content: property Screen reader compatibility](https://www.powermapper.com/tests/screen-readers/content/css-generated-content/)

<details>
<summary>MDN Content page report details</summary>

* Folder: `en-us/web/css/content`
* MDN URL: https://developer.mozilla.org/en-US/docs/Web/CSS/content
* GitHub URL: https://github.com/mdn/content/blob/main/files/en-us/web/css/content/index.md
* Last commit: https://github.com/mdn/content/commit/5f04e8cd497945960dc4d34f7a94bb7d6dce73f5
* Document last modified: 2021-09-10T08:17:56.000Z

</details>

**Comment by rachelandrew** [comment]
IE is still "certain combinations", it's also I believe still a WCAG success criteria failure: https://www.w3.org/WAI/WCAG21/Techniques/failures/F87.html so I don't think this is outdated just yet, and this is in the accessibility considerations section, as it things to take care over with regard to a11y, not rules to follow.
